### DISCUSSION TITLE
Warpcast Crashes on Android

### DISCUSSION THREAD

**Discussion by Mayakovsky** [farcasterxyz/protocol#147] [Q&A] [answered]
Trying to dig up some devs for Warpcast.  Can't find a Discord channel.

Warpcast loads on my phone and then stops drawing the UI to the screen.  So, it's loaded, running, and I can't do anything with the app.  Once in awhile it will work for about 5 minutes and then same behavior.

Getting the following error message, and I'm not sure how to parse it.   Any help greatly appreciated.

{
  "error": "\n######################################################################\n\nUnhandledFetchError\n...message: Aborted\n...name: Farcaster API Error\n...absoluteUrl: [https://client.warpcast.com/v2/client-config\n...body](https://client.warpcast.com/v2/client-config%5Cn...body): undefined\n...endpointName: getClientConfig\n...hasTimedOut: true\n...isNetworkError: false\n...method: GET\n...relativeUrl: /v2/client-config\n...resolvedTimeout: 5000\n...response: undefined\n...responseData: undefined\n...status: undefined\n...timeout: undefined\n...componentStack: \n    in Unknown\n    in Unknown\n    in InvalidateAndPurgeQueryCachesOnStartup\n    in Unknown\n    in ErrorBoundary\n    in QueryErrorResetBoundary\n    in RetryableErrorBoundary\n    in QueryClientProvider\n    in Suspense\n    in RNCSafeAreaProvider\n    in SafeAreaProvider\n    in Unknown\n    in MenuUnderlayProvider\n    in Unknown\n    in Unknown\n    in Unknown\n    in Unknown\n    in Unknown\n    in Unknown\n    in Unknown\n    in Unknown\n    in RootToastProvider\n    in ToastProvider\n    in Unknown\n    in Unknown\n    in Unknown\n    in AndroidNavigationBarColor\n    in Unknown\n    in Unknown\n    in Unknown\n    in Unknown\n    in AnalyticsEnabledProvider\n    in Unknown\n    in Unknown\n    in Unknown\n    in Unknown\n    in ErrorBoundary\n    in DebugLogsProvider\n    in ErrorHistoryProvider\n    in Unknown\n    in RNGestureHandlerRootView\n    in GestureHandlerRootView\n    in gestureHandlerRootHOC(App)\n    in RCTView\n    in Unknown\n    in RCTView\n    in Unknown\n    in AppContainer\n...isComponentError: true\n...unwrappedErrorMessage: Aborted\n...unwrappedErrorStack: Error: Aborted\n    at anonymous (address at index.android.bundle:1:387886)\n    at anonymous (address at index.android.bundle:1:386694)\n    at apply (native)\n    at anonymous (address at index.android.bundle:1:353730)\n    at _callTimer (address at index.android.bundle:1:353062)\n    at callTimers (address at index.android.bundle:1:354794)\n    at apply (native)\n    at __callFunction (address at index.android.bundle:1:209935)\n    at anonymous (address at index.android.bundle:1:208433)\n    at __guard (address at index.android.bundle:1:209370)\n    at callFunctionReturnFlushedQueue (address at index.android.bundle:1:208391)\n\n######################################################################\n",
  "localTimestamp": 1707772541171,
  "unauthed": true
}

**Comment by sds** [comment, marked as answer]
This project is for proposing changes to the Farcaster protocol, not a support page for clients built on the protocol (like Warpcast).
